<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>ABK接口测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">ABK接口测试报告</h1>
<p class='attribute'><strong>测试人员 : </strong> 余翔</p>
<p class='attribute'><strong>开始时间 : </strong> 2019-07-14 16:28:44</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:03.144606</p>
<p class='attribute'><strong>测试结果 : </strong> 共 42，通过 17，失败 21，错误 4，通过率= 40.48%</p>

<p class='description'>ABK接口测试执行结果</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 40.48% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 21 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 17 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 42 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='failClass warning'>
    <td>test_01_add.TestAdd</td>
    <td class="text-center">5</td>
    <td class="text-center">4</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',5)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='pt1_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt1_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_4</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft1_5' class='none'>
    <td class='failCase'><div class='testcase'>test_add_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_5'>失败</button>
    <div id='div_ft1_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_5'>失败</button>
    <div id='div_ft1_5' class="collapse in" align="left">
    <pre>
    
ft1_5: 测试正数与负数相加 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_01_add.py", line 44, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_01_add.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 10 != 11 : 测试正数与负数相加


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_02_sub.TestAdd</td>
    <td class="text-center">5</td>
    <td class="text-center">4</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',5)" class="detail" id='c2'>详细</a></td>
</tr>

<tr id='pt2_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt2_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_4</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft2_5' class='none'>
    <td class='failCase'><div class='testcase'>test_add_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft2_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_5'>失败</button>
    <div id='div_ft2_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft2_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_5'>失败</button>
    <div id='div_ft2_5' class="collapse in" align="left">
    <pre>
    
ft2_5: 测试正数与负数相减 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_02_sub.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_02_sub.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 5 != 15 : 测试正数与负数相减


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_03_minus.TestAdd</td>
    <td class="text-center">5</td>
    <td class="text-center">1</td>
    <td class="text-center">4</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c3',5)" class="detail" id='c3'>详细</a></td>
</tr>

<tr id='ft3_1' class='none'>
    <td class='failCase'><div class='testcase'>test_add_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_1'>失败</button>
    <div id='div_ft3_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_1'>失败</button>
    <div id='div_ft3_1' class="collapse in" align="left">
    <pre>
    
ft3_1: 测试负数与负数相乘 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: -6 != 8 : 测试负数与负数相乘


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_2' class='none'>
    <td class='failCase'><div class='testcase'>test_add_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_2'>失败</button>
    <div id='div_ft3_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_2'>失败</button>
    <div id='div_ft3_2' class="collapse in" align="left">
    <pre>
    
ft3_2: 测试负数与正数相乘 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 1 != -12 : 测试负数与正数相乘


    </pre>
    </div>
    </td>
</tr>

<tr id='pt3_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft3_4' class='none'>
    <td class='failCase'><div class='testcase'>test_add_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_4'>失败</button>
    <div id='div_ft3_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_4'>失败</button>
    <div id='div_ft3_4' class="collapse in" align="left">
    <pre>
    
ft3_4: 测试正数与正数相乘 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 8 != 15 : 测试正数与正数相乘


    </pre>
    </div>
    </td>
</tr>

<tr id='ft3_5' class='none'>
    <td class='failCase'><div class='testcase'>test_add_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft3_5'>失败</button>
    <div id='div_ft3_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft3_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft3_5'>失败</button>
    <div id='div_ft3_5' class="collapse in" align="left">
    <pre>
    
ft3_5: 测试正数与负数相乘 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_03_minus.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 2 != -26 : 测试正数与负数相乘


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_04_div.TestAdd</td>
    <td class="text-center">5</td>
    <td class="text-center">3</td>
    <td class="text-center">2</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c4',5)" class="detail" id='c4'>详细</a></td>
</tr>

<tr id='pt4_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt4_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft4_3' class='none'>
    <td class='failCase'><div class='testcase'>test_add_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_3'>失败</button>
    <div id='div_ft4_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_3'>失败</button>
    <div id='div_ft4_3' class="collapse in" align="left">
    <pre>
    
ft4_3: 测试零与零相除 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_04_div.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_04_div.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 0 != '被除数不能为0' : 测试零与零相除


    </pre>
    </div>
    </td>
</tr>

<tr id='pt4_4' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_add_4</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft4_5' class='none'>
    <td class='failCase'><div class='testcase'>test_add_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft4_5'>失败</button>
    <div id='div_ft4_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft4_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft4_5'>失败</button>
    <div id='div_ft4_5' class="collapse in" align="left">
    <pre>
    
ft4_5: 测试正数与负数相除 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_04_div.py", line 43, in test_add
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_04_div.py", line 36, in test_add
    self.assertEqual(expected, calc_res, msg=msg)
AssertionError: 2 != -6.0 : 测试正数与负数相除


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_05_register_request_case.TestRegister</td>
    <td class="text-center">7</td>
    <td class="text-center">3</td>
    <td class="text-center">4</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c5',7)" class="detail" id='c5'>详细</a></td>
</tr>

<tr id='ft5_1' class='none'>
    <td class='failCase'><div class='testcase'>test_register_1</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_1'>失败</button>
    <div id='div_ft5_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_1'>失败</button>
    <div id='div_ft5_1' class="collapse in" align="left">
    <pre>
    
ft5_1: 未注册过得手机号码注册的测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 48, in test_register
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 40, in test_register
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '注册成功' != '手机号码格式不正确'
- 注册成功
+ 手机号码格式不正确
 : 未注册过得手机号码注册的测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='pt5_2' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_register_2</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='pt5_3' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_register_3</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft5_4' class='none'>
    <td class='failCase'><div class='testcase'>test_register_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_4'>失败</button>
    <div id='div_ft5_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_4'>失败</button>
    <div id='div_ft5_4' class="collapse in" align="left">
    <pre>
    
ft5_4: 号码为空的测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 48, in test_register
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 40, in test_register
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '密码不能为空' != '手机号不能为空'
- 密码不能为空
+ 手机号不能为空
 : 号码为空的测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_5' class='none'>
    <td class='failCase'><div class='testcase'>test_register_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_5'>失败</button>
    <div id='div_ft5_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_5'>失败</button>
    <div id='div_ft5_5' class="collapse in" align="left">
    <pre>
    
ft5_5: 密码为空的测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 48, in test_register
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 40, in test_register
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '手机号码不能为空' != '密码不能为空'
- 手机号码不能为空
? ^^^
+ 密码不能为空
? ^
 : 密码为空的测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft5_6' class='none'>
    <td class='failCase'><div class='testcase'>test_register_6</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft5_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft5_6'>失败</button>
    <div id='div_ft5_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft5_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft5_6'>失败</button>
    <div id='div_ft5_6' class="collapse in" align="left">
    <pre>
    
ft5_6: 号码和密码都为空的测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 48, in test_register
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_05_register_request_case.py", line 40, in test_register
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '手机号码不能为空' != '手机号不能为空'
- 手机号码不能为空
?    -
+ 手机号不能为空
 : 号码和密码都为空的测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='pt5_7' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_register_7</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr class='errorClass warning'>
    <td>test_06_login_request_case.TestLogin</td>
    <td class="text-center">7</td>
    <td class="text-center">1</td>
    <td class="text-center">4</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c6',7)" class="detail" id='c6'>详细</a></td>
</tr>

<tr id='pt6_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_login_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft6_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_2'>错误</button>
    <div id='div_ft6_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_2'>错误</button>
    <div id='div_ft6_2' class="collapse in" align="left">
    <pre>
    
ft6_2: Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 32, in test_login
    params = params_replace.not_register(case['params'])
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\params_replace.py", line 31, in not_register
    data = self.not_exit_mobile(data)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\params_replace.py", line 14, in not_exit_mobile
    mobile = mysql.get_no_exit_mobile()
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 56, in get_no_exit_mobile
    if not self.is_exit(mobile):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 47, in is_exit
    if self.run_sql(sql, args=(mobile, )):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 21, in run_sql
    self.cursor.execute(sql, args)  # 执行sql语句
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_3'>错误</button>
    <div id='div_ft6_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_3'>错误</button>
    <div id='div_ft6_3' class="collapse in" align="left">
    <pre>
    
ft6_3: Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 32, in test_login
    params = params_replace.not_register(case['params'])
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\params_replace.py", line 31, in not_register
    data = self.not_exit_mobile(data)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\params_replace.py", line 14, in not_exit_mobile
    mobile = mysql.get_no_exit_mobile()
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 56, in get_no_exit_mobile
    if not self.is_exit(mobile):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 47, in is_exit
    if self.run_sql(sql, args=(mobile, )):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 21, in run_sql
    self.cursor.execute(sql, args)  # 执行sql语句
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 165, in execute
    while self.nextset():
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 107, in nextset
    return self._nextset(False)
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 91, in _nextset
    conn = self._get_db()
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\cursors.py", line 73, in _get_db
    raise err.ProgrammingError("Cursor closed")
pymysql.err.ProgrammingError: Cursor closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_4' class='none'>
    <td class='failCase'><div class='testcase'>test_login_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_4'>失败</button>
    <div id='div_ft6_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_4'>失败</button>
    <div id='div_ft6_4' class="collapse in" align="left">
    <pre>
    
ft6_4: 不输入账号 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 48, in test_login
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 40, in test_login
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '手机号码不能为空' != '手机号不能为空'
- 手机号码不能为空
?    -
+ 手机号不能为空
 : 不输入账号


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_5' class='none'>
    <td class='failCase'><div class='testcase'>test_login_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_5'>失败</button>
    <div id='div_ft6_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_5'>失败</button>
    <div id='div_ft6_5' class="collapse in" align="left">
    <pre>
    
ft6_5: 账号密码都不输入 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 48, in test_login
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 40, in test_login
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '手机号码或密码不能为空' != '手机号不能为空'
- 手机号码或密码不能为空
?    ----
+ 手机号不能为空
 : 账号密码都不输入


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_6' class='none'>
    <td class='failCase'><div class='testcase'>test_login_6</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_6'>失败</button>
    <div id='div_ft6_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_6'>失败</button>
    <div id='div_ft6_6' class="collapse in" align="left">
    <pre>
    
ft6_6: 错误的手机号码格式 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 48, in test_login
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 40, in test_login
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '手机号码格式不正确' != '用户名或密码错误'
- 手机号码格式不正确
+ 用户名或密码错误
 : 错误的手机号码格式


    </pre>
    </div>
    </td>
</tr>

<tr id='ft6_7' class='none'>
    <td class='failCase'><div class='testcase'>test_login_7</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft6_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft6_7'>失败</button>
    <div id='div_ft6_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft6_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft6_7'>失败</button>
    <div id='div_ft6_7' class="collapse in" align="left">
    <pre>
    
ft6_7: 错误的密码 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 48, in test_login
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 40, in test_login
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '密码错误' != '用户名或密码错误'
- 密码错误
+ 用户名或密码错误
 : 错误的密码


    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>unittest.suite._ErrorHolder</td>
    <td class="text-center">2</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">2</td>
    <td class="text-center"><a href="javascript:showClassDetail('c7',2)" class="detail" id='c7'>详细</a></td>
</tr>

<tr id='ft7_1' class='none'>
    <td class='errorCase'><div class='testcase'>TestLogin)</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft7_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft7_1'>错误</button>
    <div id='div_ft7_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft7_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft7_1'>错误</button>
    <div id='div_ft7_1' class="collapse in" align="left">
    <pre>
    
ft7_1: 错误的密码 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_06_login_request_case.py", line 53, in tearDownClass
    mysql.close_db()
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 34, in close_db
    self.conn.close()
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\connections.py", line 354, in close
    raise err.Error("Already closed")
pymysql.err.Error: Already closed


    </pre>
    </div>
    </td>
</tr>

<tr id='ft7_2' class='none'>
    <td class='errorCase'><div class='testcase'>TestRecharge)</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft7_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft7_2'>错误</button>
    <div id='div_ft7_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft7_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft7_2'>错误</button>
    <div id='div_ft7_2' class="collapse in" align="left">
    <pre>
    
ft7_2: 金额为None---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 54, in tearDownClass
    mysql.close_db()
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\scripts\mysql_class.py", line 34, in close_db
    self.conn.close()
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\pymysql\connections.py", line 354, in close
    raise err.Error("Already closed")
pymysql.err.Error: Already closed


    </pre>
    </div>
    </td>
</tr>

<tr class='failClass warning'>
    <td>test_07_recharge_request_case.TestRecharge</td>
    <td class="text-center">6</td>
    <td class="text-center">1</td>
    <td class="text-center">5</td>
    <td class="text-center">0</td>
    <td class="text-center"><a href="javascript:showClassDetail('c8',6)" class="detail" id='c8'>详细</a></td>
</tr>

<tr id='pt8_1' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_recharge_1</div></td>
    <td colspan='5' align='center'><span class="label label-success success">通过</span></td>
</tr>

<tr id='ft8_2' class='none'>
    <td class='failCase'><div class='testcase'>test_recharge_2</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft8_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft8_2'>失败</button>
    <div id='div_ft8_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft8_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft8_2'>失败</button>
    <div id='div_ft8_2' class="collapse in" align="left">
    <pre>
    
ft8_2: 充值金额在合理范围内---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 50, in test_recharge
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 42, in test_recharge
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '充值成功' != '手机号不能为空'
- 充值成功
+ 手机号不能为空
 : 充值金额在合理范围内---测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft8_3' class='none'>
    <td class='failCase'><div class='testcase'>test_recharge_3</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft8_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft8_3'>失败</button>
    <div id='div_ft8_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft8_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft8_3'>失败</button>
    <div id='div_ft8_3' class="collapse in" align="left">
    <pre>
    
ft8_3: 金额为-1---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 50, in test_recharge
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 42, in test_recharge
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '请输入范围在0到50万之间的正数金额' != '手机号不能为空'
- 请输入范围在0到50万之间的正数金额
+ 手机号不能为空
 : 金额为-1---测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft8_4' class='none'>
    <td class='failCase'><div class='testcase'>test_recharge_4</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft8_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft8_4'>失败</button>
    <div id='div_ft8_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft8_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft8_4'>失败</button>
    <div id='div_ft8_4' class="collapse in" align="left">
    <pre>
    
ft8_4: 金额为0---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 50, in test_recharge
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 42, in test_recharge
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '充值成功' != '手机号不能为空'
- 充值成功
+ 手机号不能为空
 : 金额为0---测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft8_5' class='none'>
    <td class='failCase'><div class='testcase'>test_recharge_5</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft8_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft8_5'>失败</button>
    <div id='div_ft8_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft8_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft8_5'>失败</button>
    <div id='div_ft8_5' class="collapse in" align="left">
    <pre>
    
ft8_5: 金额为1---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 50, in test_recharge
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 42, in test_recharge
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '充值成功' != '手机号不能为空'
- 充值成功
+ 手机号不能为空
 : 金额为1---测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='ft8_6' class='none'>
    <td class='failCase'><div class='testcase'>test_recharge_6</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft8_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft8_6'>失败</button>
    <div id='div_ft8_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft8_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft8_6'>失败</button>
    <div id='div_ft8_6' class="collapse in" align="left">
    <pre>
    
ft8_6: 金额为None---测试用例 不通过!
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\Bdd\venv\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 50, in test_recharge
    raise e
  File "C:\Users\Administrator\PycharmProjects\ABK_Api_Test\cases\test_07_recharge_request_case.py", line 42, in test_recharge
    self.assertEqual(expected, actual_res, msg=msg)
AssertionError: '请输入金额' != '手机号不能为空'
- 请输入金额
+ 手机号不能为空
 : 金额为None---测试用例


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>42</td>
    <td>17</td>
    <td>21</td>
    <td>4</td>
    <td>通过率：40.48%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
